cam_info:
    height: 480
    width: 640
    focal_length: 270
    K: [270.0, 0.0, 320.0,
        0.0, 270.0, 240.0,
        0.0, 0.0, 1.0]